recipes/drivelib/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "drivelib" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8e00c9981225ed7eae9f217f12242d98130aff59c2ff3e6026570589968a8fe3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # py<37
+
+requirements:
+  host:
+    - google-api-python-client
+    - google-auth
+    - google-auth-httplib2
+    - google-auth-oauthlib
+    - oauth2client
+    - pip
+    - python
```

I think you just need ``pip`` and ``python`` here
```suggestion
    - pip
    - python >=3.7
```